README.md

```diff
@@ -151,6 +151,6 @@ This data is collected from the `package.json` file in the sample application an
 
 [deploy_track_url]: https://github.com/cloudant-labs/deployment-tracker
 [cloud_foundry]: https://github.com/cloudfoundry/cli
-[getting_started]: https://www.ibm.com/watson/developercloud/doc/common/index.html
-[nlc_docs]: https://www.ibm.com/watson/developercloud/nl-classifier.html
-[sign_up]: https://console.ng.bluemix.net/registration/
+[getting_started]: https://console.bluemix.net/docs/services/watson/index.html#about
+[nlc_docs]: https://www.ibm.com/watson/services/natural-language-classifier/
+[sign_up]: https://console.bluemix.net/registration/
```

Comment:
I think we want them to get back to NLC if they have to sign up. This link would do that (assuming we want to push them to the catalog): https://console.bluemix.net/registration/?target=%2Fcatalog%2Fservices%2Fnatural-language-classifier%2F